server/chat-commands/info.ts

```diff
@@ -2770,23 +2775,25 @@ export const pages: PageTable = {
 	},
 	quotes(args, user) {
 		const room = this.requireRoom();
+		this.title = `[Quotes]`;
 		// allow it for users if they can access the room
 		if (!user.inRooms.has(room.roomid) && room.settings.isPrivate && !user.isStaff) {
 			return this.errorReply(`Access denied.`);
 		}
 		let buffer = `<div class="pad">`;
+		buffer += `<button style="float:right;" class="button" name="send" value="/join view-quotes-${room.roomid}"><i class="fa fa-refresh"></i> Refresh</button>`;
 		if (!room.settings.quotes?.length) {
 			return `${buffer}<h2>This room has no quotes.</h2></div>`;
 		}
 
-		buffer = `${buffer}<h2>Quotes on ${room.title}: (${room.settings.quotes.length})</h2>`;
+		buffer += `<h2>Quotes for ${room.title} (${room.settings.quotes.length}):</h2>`;
 		for (const entry of room.settings.quotes) {
 			const index = room.settings.quotes.indexOf(entry) + 1;
 			const {quote, userid, date} = entry;
-			buffer += `<div class="infobox">${index}: ${Chat.collapseLineBreaksHTML(Chat.formatText(quote))}`;
-			buffer += `<br /> Added by ${userid} on ${Chat.toTimestamp(new Date(date), {human: true})}`;
+			buffer += `<div class="infobox">${index}: ${Chat.formatText(quote).replace(/\n/g, '<br />')}`;
```

Comment:
Why not just collapse the line breaks when the quote is added? The only places where the quote is shown, it would be collapsed anyway, so it'd save some time to collapse once rather then in randquote AND in the page, imo.

Reply:
This change makes it so it no longer collapses on the page because the way it currently displays multi-line quotes is very cluttered, and even if i collapsed while adding, I would end up just doing the same exact functionality but with `.replace(/&amp;#10;/g, '<br />')` instead of what's currently used. Additionally, formatting upon adding would make the modlog entry and mod action look extremely ugly (`(user added a quote: a<b>bolded text</b>)`) for example